app/javascript/application.js

```diff
@@ -3,12 +3,18 @@ import "@hotwired/turbo-rails"
 import "./controllers"
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { fetchRandomGreeting } from '../redux/actions';
+import store from '../redux/store';
+import App from '../components/App';
```

Comment:
- [ ] I was unable to set up your app. You are missing the `react-redux` package in your `package.json` file. Kindly install it: `npm i react-redux`.

  ![kdieo](https://github.com/djo1975/Hello-World/assets/36542986/30b699d3-c0cc-4d20-81f6-7f1835553e15)

- [ ] Your import path is also wrong on lines 7 to 9. It should be `./`, not `../`. Kindly fix that.

  ![mkkdd](https://github.com/djo1975/Hello-World/assets/36542986/82013148-1117-493f-8a05-36abb1555205)